sdks/java/extensions/zetasketch/src/main/java/org/apache/beam/sdk/extensions/zetasketch/HllCount.java

```diff
@@ -346,7 +346,9 @@ private Extract() {}
                   @ProcessElement
                   public void processElement(
                       @Element byte[] sketch, OutputReceiver<Long> receiver) {
-                    receiver.output(HyperLogLogPlusPlus.forProto(sketch).result());
+                    Long result =
+                        (sketch == null) ? 0L : HyperLogLogPlusPlus.forProto(sketch).result();
```

Comment:
Would it make sense to document the nullability of the sketch param somewhere -- method doc or @nullable annotation on param, ...